rewrite-gradle/src/test/java/org/openrewrite/gradle/UpgradeDependencyVersionTest.java

```diff
@@ -986,6 +986,29 @@ void disallowDowngrade() {
         );
     }
 
+    @Test
+    void retainLatestReleaseOrLatestIntegrationIfUsed() {
+        rewriteRun(
+          spec -> spec.recipe(new UpgradeDependencyVersion("org.projectlombok", "lombok", "1.18.*", null)),
```

Comment:
Adding the context here that this is [coming from](https://github.com/openrewrite/rewrite-migrate-java/blob/e4e2ecd60ed8f9e013b43b0168f5497471bd59a3/src/main/resources/META-INF/rewrite/lombok.yml#L41-L51) in `org.openrewrite.java.migrate.lombok.UpdateLombokToJava11`.